e2e/cypress/integration/files_and_attachments/upload_files_spec.js

```diff
@@ -52,4 +64,26 @@ describe('Upload Files', () => {
             });
         });
     });
+
+    it('MM-T12 Loading indicator when posting images', () => {
+        const filename = 'huge-image.jpg';
+
+        // # Post an image in center channel
+        cy.get('#centerChannelFooter').find('#fileUploadInput').attachFile(filename);
+        cy.get('.post-image').should('be.visible');
+        cy.postMessage('{enter}');
+
+        // # Login as otherUser
+        cy.apiLogin(otherUser);
```

Comment:
Better to reload the page after login and verify that the `.image-loading__container` is visible.